test/controllers/ReportableEventsControllerSpec.scala

```diff
@@ -113,16 +114,26 @@ class ReportableEventsControllerSpec extends UnitSpec with ERSFakeApplicationCon
     "direct to ers errors page if fetching ersMetaData throws exception" in {
       val controllerUnderTest = buildFakeReportableEventsController(ersMetaDataRes = false)
       val result = controllerUnderTest.updateErsMetaData(ersRequestObject)(Fixtures.buildFakeUser, Fixtures.buildFakeRequestWithSessionIdCSOP("GET"), hc)
+      result match {
```

Comment:
Is the match necessary here (and below)? I feel like just having lines 119 & 120 does the same thing

Reply:
```val result = controllerUnderTest.updateErsMetaData(ersRequestObject)(Fixtures.buildFakeUser, Fixtures.buildFakeRequestWithSessionIdCSOP("GET"), hc).asInstanceOf[Future[Result]]
      status(result) shouldBe 500
      contentAsString(result) shouldBe contentAsString(Future.successful(controllerUnderTest.getGlobalErrorPage))
    }
    ```

Reply:
As discussed, I'll change this from the pattern match and call `asInstanceOf` instead